examples/diamond/psp22_2_facet/lib.rs

```diff
@@ -0,0 +1,51 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![feature(min_specialization)]
+
+#[brush::contract]
+pub mod my_psp22_2_facet {
+    use brush::{
+        contracts::{
+            ownable::*,
+            psp22::*,
+        },
+        traits::ZERO_ADDRESS,
+    };
+    use ink_lang::codegen::Env;
+    use ink_prelude::vec::Vec;
+    use ink_storage::traits::SpreadAllocate;
+
+    #[ink(storage)]
+    #[derive(Default, SpreadAllocate, PSP22Storage, OwnableStorage)]
+    pub struct PSP22Facet2 {
+        #[PSP22StorageField]
+        psp22: PSP22Data,
+        #[OwnableStorageField]
+        ownable: OwnableData,
+    }
+
+    impl PSP22 for PSP22Facet2 {
```

Comment:
Maybe let's name it `psp22_faucet_v2`. And let's name `psp22_faucet` -> `psp22_faucet_v1`